src/main/scala/mesosphere/marathon/event/http/HttpEventStreamServlet.scala

```diff
@@ -0,0 +1,48 @@
+package mesosphere.marathon.event.http
+
+import javax.inject.{ Inject, Named }
+import javax.servlet.http.HttpServletRequest
+
+import akka.actor.ActorRef
+import mesosphere.marathon.ModuleNames
+import mesosphere.marathon.event.http.HttpEventStreamActor._
+import org.eclipse.jetty.servlets.EventSource.Emitter
+import org.eclipse.jetty.servlets.{ EventSource, EventSourceServlet }
+
+/**
+  * Handle a server side event client stream by delegating events to the stream actor.
+  * @param request the initial http request.
+  * @param httpStreamActor the stream actor.
+  */
+class HttpEventSSEHandle(request: HttpServletRequest, httpStreamActor: ActorRef)
+    extends EventSource with HttpEventStreamHandle {
+
+  private var emitter: Option[Emitter] = None
+
+  override def onOpen(emitter: Emitter): Unit = {
+    this.emitter = Some(emitter)
```

Comment:
This is the glue between the servlet that creates the handle, which than sends messages to the actor.